ecommerce/core/models.py

```
@@ -552,7 +551,7 @@ def is_verified(self, site):
         except (ConnectionError, SlumberBaseException, Timeout):
             msg = 'Failed to retrieve verification status details for [{username}]'.format(username=self.username)
             log.exception(msg)
-            raise VerificationStatusError(msg)
+            return False
```

Maybe it would be better to catch the exception where it's used? It makes more sense to raise an exception here instead of assuming that the user is verified if the endpoint times out.

Either that or keep this and change the docstring on this method.